Add the ability to specify a different buf.gen.yaml file (#107)
andrewparmet/buf-gradle-plugin: README.md

}
```

> You can optionally specify a different `buf.gen.yaml` file as well:

Reviewer: This should be under a separate heading from "Generating Dependencies" and added to the table of contents. Possibly called "Further Configuration".

Reviewer: Not something for this PR: it seems like a potential normal use case is to want several invocations of `buf generate`, each with a different template. I'll look into that use case separately.

Ensure you have an up-to-date `buf.lock` file generated by `buf mod update` or this generation will fail.

#### Further Configuration

> By default, the `bufGenerate` task will attempt to read a `buf.gen.yaml` template file from the root directory. You can override the location of the template file as such:

Reviewer: Tiny nit: needless words
```suggestion
By default `bufGenerate` will read the `buf.gen.yaml` template file from the project root directory. You can override the location of the template file:
```
